### PR TITLE
Hts 222

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: run_selenium_system_test.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ tags    - Space separated list of tags. Only run tests with the given
           tags. Not specifying any tags will run all tests.
```
Comment (MarcusQA) [inline run_selenium_system_test.sh]: On line 6 of  run_selenium_system_test.sh can we include [zap-chrome], please?

File: project/MicroService.scala [hunk 2]
```diff
@@ -22,6 +22,8 @@ trait MicroService {
   lazy val plugins: Seq[Plugins] = Seq.empty
   lazy val playSettings: Seq[Setting[_]] = Seq.empty
 
+  val hmrcRepoHost = java.lang.System.getProperty("hmrc.repo.host", "https://nexus-dev.tax.service.gov.uk")
```
Comment (MarcusQA) [inline project/MicroService.scala]: Is this used if you're running the zap test locally?

Comment (Jackie1988) [reply project/MicroService.scala]: I don't think this is used at all atm, I will check and then remove.

File: run_selenium_system_test.sh (hunk 1 above)
Comment (Jackie1988) [reply run_selenium_system_test.sh]: Yeah sure

**Pull request merged**